resources/css/app.css

```diff
@@ -1,3 +1,120 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+.section-padding {
```

Comment:
It is necessary to create this all classes ?

Reply:
Do you think it would be better to paste tailwind classes in every place?

Reply:
Of course in every places it wouldn't be possible, but I think you can replace most css code from this file by tailwind classes.

Reply:
okay, switched to tailwind where its possible